components/layout/Footer.tsx

```diff
@@ -58,6 +58,10 @@ const footerNav = [
         label: 'Support',
         link: '/docs/support',
       },
+      {
+        link: '/press',
```

Comment:
Shouldn't this be `/media` as well?

```suggestion
        link: '/media',
```

Reply:
resolved